js/tests/unit/dropdown.js

```diff
@@ -84,10 +84,11 @@ $(function () {
     $(dropdownHTML).appendTo('#qunit-fixture')
     var $dropdown = $('#qunit-fixture').find('[data-toggle="dropdown"]').bootstrapDropdown()
     var $button = $('button[data-toggle="dropdown"]')
+    $button[0].focus()
     // Key escape
-    $button.trigger('focus').trigger($.Event('keydown', {
-      which: 27
-    }))
+    var keydown = new Event('keydown')
```

Comment:
using `new Event` isn't supported on IE 10 and IE 11, you should use jQuery event system

Reply:
I've already dropped the patch from this branch. If @patrickhlauke wants, we should have a separate PR.